contracts/Removal.sol

```diff
@@ -499,6 +446,58 @@ contract Removal is
     return id.decodeRemovalIdV0();
   }
 
+  /**
+   * @dev Transfers `amount` tokens of token type `id` from `from` to `to`.
+   *
+   * Emits a {TransferSingle} event.
```

Comment:
From the todo list:

> Remove natspec comments that use syntax including {...} (e.g., See {[https://docs.openzeppelin.com/contracts/4.x/api/token/erc20#ERC20Permit}](https://docs.openzeppelin.com/contracts/4.x/api/token/erc20#ERC20Permit%7D)) as this syntax is only valid for OpenZeppelins custom internal docgen

Reply:
that's my bad, I started doing that in LockedNORI not realizing that the curly braces were OZ private sauce.